tools/Dockerfile.citadel

```diff
@@ -20,62 +20,62 @@ RUN scripts/enable_gcc8.sh
 
 COPY scripts/build_ign.sh scripts/build_ign.sh
 
-RUN scripts/build_ign.sh ignitionrobotics ign-cmake ign-cmake2 n \
-      $IGN_VERSION_DATE \
-      $IGN_VERSION_PASSWORD
+# RUN scripts/build_ign.sh ignitionrobotics ign-cmake ign-cmake2 n \
+#       $IGN_VERSION_DATE \
+#       $IGN_VERSION_PASSWORD; exit 0
```

Comment:
Perhaps we should mention why `ign-cmake` (and `ign-tools`) are commented out, so that we don't forget if we have to come back to this later. Also, would it be worth removing the old `ign-cmake` and `ign-tools` segments from this file instead of commenting them out, since we can always refer to the git history? (this comment also applies to the changes made in `Dockerfile.dome`)

Reply:
Good idea, added notes in 95a837a